core/ui/PageTemplate/sidebar.tid

```diff
@@ -6,6 +6,10 @@ tags: $:/tags/PageTemplate
 $:/config/SideBarSegments/Visibility/$(listItem)$
 \end
 
+<div class="tc-sidebar-scrollable">
```

Comment:
It seems confusing to be reusing the tc-sidebar-scrollable class, is there a reason for this?

Reply:
Hi @Jermolene, it's indeed confusing. The reason is that the `<div class="sidebar-resizer">` here needs to be wrapped by the sidebar-scrollable div but that div needs to have `overflow: visible` so that the sidebar-resizer can be moved up if the `$:/themes/tiddlywiki/vanilla/metrics/storytop` is set to something bigger than 0. The sidebar-scrollable is a `<$scrollable>` widget with `<style="overflow: auto;">` and that doesn't allow me to do so. It's a quick hack. I'm playing with different approaches right now and this one here is not the best possible one